File: pilgrim/app/assets/javascripts/word.js
$(document).ready(function(){
  $('.class1').on('click', function(){
    var current = this;
    var word = ($(current).html());
    $.getJSON('https://www.googleapis.com/language/translate/v2?key=AIzaSyALN7om8pcP6n5BhSB0v9K23KQB4B1mefo&q=' + word + '&source=en&target=es', function(data) {
      var replacement = (data.data["translations"][0]["translatedText"]);
      console.log(current);
      console.log(replacement);
      $(current).html(replacement);
    });
  });
});

File: app/models/word.rb
class Word < ActiveRecord::Base
  attr_accessible :entry

  # def translate
  #   data = open('https://www.googleapis.com/language/translate/v2?key=AIzaSyALN7om8pcP6n5BhSB0v9K23KQB4B1mefo&q=' + word.entry + '&source=en&target=sp')
  #   puts data
  # end
end
